docs/pages/cheatsheet.adoc

```diff
@@ -323,25 +294,51 @@ No syntax change.
 }
 ----
 
-=== Transfer primitive field
+=== Transfer initializer
+
+The initializer is automatically called when a transfer object is created or selected. It can be used to set the values of the transient fields.
 
 *Syntax:*
 
 [subs="quotes"]
 ----
-[purple]#**field**# &#x5B;[purple]##**required**##] &lt;primitive> &lt;name> &#x5B;([purple]##**<=**##|[purple]##**<=>**##) &lt;expression>] [purple]#**;**#
+[purple]#**initializer**# [purple]#**;**#
+----
+
+=== Transfer destructor
+
+Before the transfer object instance is deleted, the system calls the `destructor`. The destructor is a special action that is used to delete mapped entity or invoke other destruct actions. Transfer objects with no destructor can not be created. The default behavior of the destructor is to delete the mapped entity.
+
+*Syntax:*
+
+[subs="quotes"]
+----
+[purple]#**destructor**# [purple]#**;**#
 ----
 
-*Old syntax:*
+=== Transfer primitive field
+
+A transfer object may contain data descriptions called primitive fields. A primitive field can store only a single primitive value.
+
+*Syntax:*
 
 [subs="quotes"]
 ----
-[purple]#**field**# &#x5B;[purple]##**required**##] &lt;primitive> &lt;name> &#x5B;([purple]##**reads**##|[purple]##**maps**##) &lt;expression>] [purple]#**;**#
+[purple]#**field**# &#x5B;[purple]##**required**##] &lt;primitive> &lt;name> &#x5B;([purple]##**<=**##|[purple]##**<=>**##) &lt;expression>] [purple]#**;**#
 ----
 
+*Annotations:*
+
+[options="header", cols="1,2"]
+|====================
+|Annotation | Meaning
+|@Lazy      | Defer fetching the object(s) as long as it is possible.
```

Comment:
What does 'as long as it is possible' mean? Fetch when it is "accessed"?

Reply:
By telling "as long as it is possible" one could interpret this as something like fetching only if its "available" or "defined"

Reply:
rephrased